app/api/dao/user.py

```diff
@@ -3,7 +3,8 @@
 from app.api.email_utils import confirm_token
 from app.database.models.user import UserModel
 from app.utils.validation_utils import is_email_valid
-
+from app.utils import aws_upload
+import os
 
 class UserDAO:
```

Comment:
Add one more blank line before class definition. Refer: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
@ramitsawhney27 @agarwalrounak I'm little confused about which linting tool we are using. It's pep8 , flake8 or pylint ?

Reply:
pylint

Reply:
@ramitsawhney27 Thanks for the info :)